SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestGridLBStrategy.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "SpatialFunctionalTestGridLBStrategy.h"
+#include "GameFramework/Actor.h"
+#include "SpatialFunctionalTestWorkerDelegationComponent.h"
+
+USpatialFunctionalTestGridLBStrategy::USpatialFunctionalTestGridLBStrategy()
+{
+	Rows = 2;
+	Cols = 2;
+	InterestBorder = 500.0f;
```

Comment:
Do you temporarily want to make the interest area larger until the GDK fix is available?

Reply:
The idea is for this to be a base class that you can extend, atm the base class sets InterestBorder of 0.0f, so I'd probably prefer to keep the base class value for consistency. I'll remove that InterestBorder override